packages/web-forms/src/index.ts

```diff
@@ -2,13 +2,6 @@ import { webFormsPlugin } from './WebFormsPlugin';
 import OdkWebForm from './components/OdkWebForm.vue';
 import { POST_SUBMIT__NEW_INSTANCE } from './lib/constants/control-flow.ts';
 
-import '@fontsource/roboto/300.css';
-import './assets/css/icomoon.css';
-import './themes/2024-light/theme.scss';
-
-// TODO/sk: Purge it - using postcss-purgecss
-import 'primeflex/primeflex.css';
-
```

Comment:
I didn't see these being effective (no loaded/bundled) Looking at the config when building and the /dist, these are taken from demo bundle.

Reply:
Now I can test this it, after using web forms as a plugin in a blank Vue project. 
This is resolved.